scss/titles.scss

```diff
@@ -0,0 +1,25 @@
+@import 'colors';
+
+.title {
+  max-width: 400px;
+  color: $dark-black;
+  font-size: 30px;
+  margin: 50px;
+  position: relative;
+  top: 0px;
```

Comment:
You shouldn't need `top, left, right or bottom` positions on relative elements, you should use `margin` for that porpouse